### ISSUE TITLE
Invalid fileprivate replacements

### ISSUE THREAD

**Issue by jshier** [nicklockwood/SwiftFormat#837]
SwiftFormat 0.47.9 makes two invalid `fileprivate` -> `private` conversions in `Request.swift` when formatting Alamofire.

```swift
    fileprivate var uploadProgressHandler: (handler: ProgressHandler, queue: DispatchQueue)? {
        get { mutableState.uploadProgressHandler }
        set { mutableState.uploadProgressHandler = newValue }
    }
```

is formatted into 

```swift
    private var uploadProgressHandler: (handler: ProgressHandler, queue: DispatchQueue)? {
        get { mutableState.uploadProgressHandler }
        set { mutableState.uploadProgressHandler = newValue }
    }
```

and 

```swift
    @Protected
    fileprivate var validators: [() -> Void] = []
```

is formatted into 

```swift
    @Protected
    private var validators: [() -> Void] = []
```

**Comment by nicklockwood** [comment]
@jshier thanks for reporting it, I'll take a look.

**Comment by nicklockwood** [comment]
@jshier AFAICT, this is only a problem for the `validators` property, correct? Changing `uploadProgressHandler` to private does not appear to cause a compilation failure.

**Comment by nicklockwood** [comment]
@jshier fixed in 0.47.10
